pkgs/tools/compression/lziprecover/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "lziprecover";
+  version = "1.22";
+
+  src = fetchurl {
+    url = "mirror://savannah/lzip/lziprecover/${pname}-${version}.tar.gz";
+    sha256 = "sha256-/ZWKCXX3cpxE87eE5WaJH3NsPcaDdNvSFJ7mkqFtCGI=";
+  };
+
+  configureFlags = [
+    "CPPFLAGS=-DNDEBUG"
+    "CFLAGS=-O3"
+    "CXXFLAGS=-O3"
+  ] ++ lib.optional (stdenv.hostPlatform != stdenv.buildPlatform)
+    "CXX=${stdenv.cc.targetPrefix}c++";
```

Comment:
Can we apply this uncondtional?

Reply:
I guess we can, I modified lzip too, from which this idiom was copied.